src/frontend/web_application/test/functional/features/22-compose-spec.js

```diff
@@ -5,11 +5,11 @@ describe('Compose new message', () => {
   const locale = 'en';
   const __ = key => ({
     fr: {
-      Compose: 'Écrire',
+      Compose: 'ÉCRIRE',
```

Comment:
French should'nt be useful anymore since I've correctly configure chrome to start in english